website/src/data/users.tsx

```diff
@@ -808,6 +808,15 @@ const Users: User[] = [
     source: null,
     tags: ['design'],
   },
+  {
+    title: 'Molecule',
+    description:
+      'The Molecule is a lightweight Web IDE UI framework which is built with React.js and inspired by the VSCode.',
```

Comment:
```suggestion
      'Molecule is a lightweight Web IDE UI framework built with React.js and inspired by VSCode.',
```

"The Molecule" sounds weird to me, isn't it better to remove those "The" ?

Reply:
Yes, it does sound weird. I thought for a second to do the edit but then decided that users are responsible for their own marketing anyways 😄